mxcube3/ui/containers/BeamlineSetupContainer.jsx

```diff
@@ -4,7 +4,8 @@ import { connect } from 'react-redux';
 import { Row, Col, Table, Popover } from 'react-bootstrap';
 import PopInput from '../components/PopInput/PopInput';
 import BeamlineActions from './BeamlineActionsContainer';
-import InOutSwitch from '../components/InOutSwitch/InOutSwitch';
+import InOutSwitch2 from '../components/InOutSwitch2/InOutSwitch2';
```

Comment:
InOutSwitch2 and InOutSwitch have been merged to InOutSwitch. The idea was to only use InOutSwitch